packages/extra/etcd/README.md

```diff
@@ -0,0 +1,10 @@
+# Etcd-cluster
+
+## Parameters
+
+### Common parameters
+
+| Name       | Description             | Value |
+| ---------- | ----------------------- | ----- |
+| `size`     | Persistent Volume size  | `4Gi` |
```

Comment:
I think 1GB is more than enough, but with double reserv 2GB should be enough. We do not want to eat everything that is on the servers with our stack. Overhead should be minimized. We will fight for every byte.

Reply:
I'd like to keep 4Gi by default for such critical component as etcd.
In oud case it might be multi-tenant, eg. when user use same cluster for multiple Kubernetes clusters

Reply:
Given the fact that it does not reserve in any way, but only sets a limit, I think you can leave 4 GB for now. But it would be good for 2GB tenant to make even 1GB enough for small clusters.